### PR TITLE
KEYCLOAK-7969 - SAML users should not be identified by SAML:NameID

### PR DESCRIPTION
**Pull request by dteleguin**

### CODE REVIEW

File: server_admin/topics/identity-broker/saml.adoc [hunk 1]
```diff
@@ -30,6 +30,13 @@ You must define the SAML configuration options as well.  They basically describe
 |NameID Policy Format
 |Specifies the URI reference corresponding to a name identifier format. Defaults to `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`.
 
+|Principal
+|Specifies which part of SAML assertion will be used to identify and track external user identities. Can be either Subject NameID or SAML attribute (either by name or by friendly name).
+ The use of Subject NameID is discouraged in favor of SAML attributes; see https://kantarainitiative.github.io/SAMLprofiles/saml2int.html[ SAML V2.0 Deployment Profile for Federation Interoperability], 3.1.3. Subject Identification for more details.
```
Comment (hmlnarik) [inline server_admin/topics/identity-broker/saml.adoc]: Please remove this line.

**Review by hmlnarik — DISMISSED** [review]

File: server_admin/topics/identity-broker/saml.adoc [hunk 2]
```diff
@@ -30,6 +30,12 @@ You must define the SAML configuration options as well.  They basically describe
 |NameID Policy Format
 |Specifies the URI reference corresponding to a name identifier format. Defaults to `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`.
 
+|Principal Type
+|Specifies which part of SAML assertion will be used to identify and track external user identities. Can be either Subject NameID or SAML attribute (either by name or by friendly name).
```
Comment (andymunro) [inline server_admin/topics/identity-broker/saml.adoc]: "the" SAML assertion

**Review by andymunro — CHANGES_REQUESTED** [review]
One minor change.

File: server_admin/topics/identity-broker/saml.adoc (hunk 2 above)
Comment (dteleguin) [reply server_admin/topics/identity-broker/saml.adoc]: done

**Review by hmlnarik — APPROVED** [review]

**Pull request merged**